modules/libcircllhist/0.3.2/patches/add_build_file.patch

```diff
@@ -0,0 +1,23 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,20 @@
++licenses(["notice"])  # Apache 2
++
++config_setting(
++    name = "windows_x86_64",
++    values = {"cpu": "x64_windows"},
++)
++
++cc_library(
++    name = "libcircllhist",
```

Comment:
The library artifact will be called `liblibcirclhist.so`. You could instead call the private target `circlhist` and then add an alias to it with name `libcirclhist`.